src/tests/ie_test_utils/common_test_utils/data_utils.cpp

```diff
@@ -202,7 +205,132 @@ size_t byte_size(const InferenceEngine::TensorDesc &tdesc) {
     auto dims = tdesc.getDims();
     return prc.size() * std::accumulate(std::begin(dims), std::end(dims), (size_t)1, std::multiplies<size_t>());
 }
+OPENVINO_SUPPRESS_DEPRECATED_END
+
+
+void fill_data_with_broadcast(ov::Tensor& tensor, ov::Tensor& values) {
+    constexpr size_t MAX_N_DIMS = 7;  // Suppose it's enough
+
+    IE_ASSERT(tensor.get_element_type() == values.get_element_type());
+
+    auto values_dims = values.get_shape();
+    auto tensor_dims = tensor.get_shape();
+    auto n_dims = tensor_dims.size();
+    IE_ASSERT(values_dims.size() <= n_dims);
+    IE_ASSERT(n_dims <= MAX_N_DIMS);
+
+    SizeVector src_dims(MAX_N_DIMS, 1);
```

Comment:
Let's use `ov::Shape` instead of `SizeVector`

Reply:
done